circuits/verify_nullifier.circom

```diff
@@ -258,13 +261,14 @@ template a_div_b_pow_c(n, k) {
 }
 
 template sha256_12_coordinates(n, k) {
-    signal input coordinates[12][k];
+    signal input pk_compressed[33];
+    signal input coordinates[10][k];
     signal input preimage_bit_length;
     signal output out[256];
 
     // compress coordinates
-    component compressors[6];
```

Comment:
Why did you change it and why is it magic variable?

Reply:
we only need 5 compressors as pk is already compressed